### DISCUSSION TITLE
Чем отличаются "Пользовательский модуль базовый" от "Пользовательский модуль расширенный"?

### DISCUSSION THREAD

**Discussion by alphagamma5** [YetiForceCompany/YetiForceCRM#15426] [How to] [answered]
Чем отличаются "Пользовательский модуль базовый" от "Пользовательский модуль расширенный"?

**Comment by vovpff** [comment, marked as answer]
В базовом просто карточка модуля с некоторым количесивом полей, в расширенном внизу/справа есть еще перечень товаров/услуг
